src/main/java/com/github/tonivade/resp/protocol/RedisSerializer.java

```diff
@@ -115,6 +135,7 @@ private void growBuffer(int len) {
       byte[] array = new byte[buffer.position()];
       ((Buffer) buffer).rewind();
       buffer.get(array);
+      buffer.clear();
```

Comment:
Here judge buffer's size is 1024, if not, renew buffer by ` ByteBuffer.allocate(INITIAL_CAPACITY)`, because the buffer maybe have grown. What do you think of it?

Reply:
I think you are right, I'm going to renew the buffer